jota/src/test/java/org/iota/jota/utils/InputValidatorTest.java

```diff
@@ -68,7 +66,7 @@ public void shouldIsArrayOfHashes() {
 
     @Test
     public void shouldIsArrayOfTrytes() {
-        assertEquals(InputValidator.isArrayOfTrytes(new String[]{TEST_TRYTES, TEST_TRYTES}), true);
+        assertTrue(InputValidator.isArrayOfTrytes(new String[]{TEST_TRYTES, TEST_TRYTES}));
```

Comment:
Codacy found an issue: [JUnit assertions should include a message](https://app.codacy.com/manual/Iota-java/iota-java/pullRequest?prid=6319461)